components/TaskCard.tsx

```
@@ -1,72 +1,57 @@
 
   const handleClick = (task: Task) => {
     setActiveTask(task)
   }
 
   return (
     <div
-      className={`flex justify-between items-center px-6 py-3 mt-1 mx-4 border border-[#D0D5DD] rounded-lg ${className}`}
+      className={`mx-4 mt-1 flex items-center justify-between rounded-lg border border-[#D0D5DD] px-6 py-3 ${className}`}
       data-testid={`task-${task.id}`}
       onClick={() => handleClick(task)}
     >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)